src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -80,8 +80,32 @@ void remove_redundant_reorders::run(program& p) {
             auto i8_u8_input = input.get_output_layout().data_type == data_types::i8 ||
                                input.get_output_layout().data_type == data_types::u8;
             auto quantize_user = has_quantize_user(node);
+            auto only_change_dtype = [](program_node& input, reorder_node& node) {
+                auto input_layout = input.get_output_layout();
+                for (auto usr : node.get_users()) {
+                    auto usr_layout = usr->get_output_layout();
+                    if (input_layout.data_type == usr_layout.data_type || input_layout.format != usr_layout.format) {
+                        return false;
+                    }
+                }
+                return true;
```

Comment:
How about add a new property in reorder prim presenting "only_type_conversion", when it is created by Conversion?

Reply:
It's a good idea. But, the targets are not only the reorder from Convert operation.